app/javascript/packs/disableUnchangedButton.js

```diff
@@ -0,0 +1,35 @@
+document.getElementById('follows_update_form').addEventListener('submit', checkChanged);
```

Comment:
could you rename this file in something self explanatory?

I worry that `disableUnchangedButton` doesn't really communicate that it's related to the dashboard tags. Maybe something like `dashboardTagsDisableUnchangedButtons.js` ?